tcms/testplans/static/testplans/js/get.js

```diff
@@ -19,26 +19,39 @@ $(document).ready(function() {
                 testCases.not_confirmed.push(data[i]);
             }
         }
-        drawConfirmedTestCases();
+        drawConfirmedTestCases(testCases.confirmed);
     });
 
-    function drawConfirmedTestCases() {
-        var container = $('#confirmed-testcases'),
-            noCasesTemplate = $('#no_test_cases'),
-            testCaseRow = $('#test_case_row')[0],
-            testCaseSummary = $(testCaseRow.content).find('.list-view-pf-main-info a.testcase')[0];
-
-        if (testCases.confirmed.length > 0) {
-            for (var i = 0; i < testCases.confirmed.length; i++) {
-                var testCase = testCases.confirmed[i];
-                testCaseSummary.href = `/case/${testCase.id}/`;
-                testCaseSummary.innerHTML = `TC-${testCase.id}: ${testCase.summary}`;
-                container.append($(testCaseRow).html());
-            }
-        } else {
-            container.append(noCasesTemplate[0].innerHTML);
-        }
+});
 
-        treeViewBind();
+function drawConfirmedTestCases(testCases) {
+    var container = $('#confirmed-testcases'),
+        noCasesTemplate = $('#no_test_cases'),
+        testCaseRow = $('#test_case_row')[0],
+        testCaseSummary = $(testCaseRow.content).find('.list-view-pf-main-info a.testcase')[0];
+
+    if (testCases.length > 0) {
+        for (var i = 0; i < testCases.length; i++) {
+            var testCase = testCases[i];
+            testCaseSummary.innerHTML = getTestCaseRowContent(testCase);
+            container.append($(testCaseRow).html());
+        }
+    } else {
+        container.append(noCasesTemplate[0].innerHTML);
     }
-});
+
+    treeViewBind();
```

Comment:
IMO this can stay in .ready function. Once you start displaying non-confirmed test cases they will also be using a tree view.